subprojects/logging/src/integTest/groovy/org/gradle/internal/logging/console/taskgrouping/ConsoleCompositeBuildGroupedTaskFunctionalTest.groovy

```diff
@@ -14,15 +14,9 @@
  * limitations under the License.
  */
 
-package org.gradle.internal.logging
+package org.gradle.internal.logging.console.taskgrouping
 
-import org.gradle.integtests.fixtures.AbstractConsoleFunctionalSpec
-import org.gradle.internal.SystemProperties
-import org.junit.Ignore
-
-@Ignore("Flaky - either fails or locks up on Windows")
```

Comment:
❓ Was flakiness addressed by splitting up the assertions below or something else?